### PR TITLE
fix tdp resume command

### LINKED ISSUE #361: [cli] `tdp resume` command is not working anymore
`tdp resume` command hasn't been adapted since the merged of #321. Hence, it doesn't work anymore.

The command is still looking for the latest operation saved in the database for a deployment whereas all operations are saved before running now. `resume` needs to look for the `FAILURE` operation instead of the last one saved.

### PR DESCRIPTION
**Pull request by PaulFarault**
#### Which issue(s) this PR fixes

Fixes #361 

#### Additional comments

ORM model may be improved to automatically query the operation logs ordered by `order`. This could be done in another issue.

#### Agreements

- [X] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [X] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Review by gonzaloetjo — APPROVED** [review]

**Review by sergkudinov — COMMENTED** [review]
Why do you put all the held operations to targets? If you don't, absence of targets doesn't change anything, no?

File: tdp/core/deployment/deployment_plan.py [hunk 1]
```diff
@@ -192,60 +192,47 @@ def from_failed_deployment(dag: Dag, deployment_log: DeploymentLog):
...
+                for operation in previous_operations[failed_operation_id:]
+            ]
+            operations_to_resume = [
+                Operation(operation_name)
+                for operation_name in operations_names_to_resume
+            ]
 
         new_deployment_log = DeploymentLog(
-            targets=remaining_operation_names,
+            targets=operations_names_to_resume,
```
Comment (sergkudinov) [inline tdp/core/deployment/deployment_plan.py:229]: Why do we put all the held operations to targets? If it is not, absence of targets doesn't change anything, no?

Comment (PaulFarault) [reply tdp/core/deployment/deployment_plan.py:229]: For logging purpose, this allow use to know what is resuming directly from the deployment list (I did not change this behavior).

I might suggest some improvements for the database schema later, but that's not a priority as it works for now.

**Review by rpignolet — APPROVED** [review]

**Comment by PaulFarault** [comment]
Forced pushed after rebase on matser.

**Pull request merged**